### PR TITLE
Adding Typographies: Text, Title & Paragraph

### PR DESCRIPTION
**Pull request by saidziani**
Missing `onChange` on **Text** and **Paragraph**

### CODE REVIEW

File: Feliz.AntDesign/Ant.fs [hunk 1]
```diff
@@ -34,4 +34,16 @@ type Ant =
     static member inline icon (children: #seq<ReactElement>) =
         ofImport "Icon" "antd" children
     static member inline icon (properties: IReactProperty list) =
-        Interop.reactApi.createElement(import "Icon" "antd", createObj !!properties)
\ No newline at end of file
+        Interop.reactApi.createElement(import "Icon" "antd", createObj !!properties)
+    static member inline text (children: #seq<ReactElement>) =
+        ofImport "Typography.Text" "antd" children
```
Comment (Zaid-Ajaj) [inline Feliz.AntDesign/Ant.fs]: When using `ofImport`, the `children` parameter has to be the last one and before it is an empty list for the properties:
```fs
ofImport "Typography.Text" "antd" [ ] children
```

File: Feliz.AntDesign/Ant.fs [hunk 2]
```diff
@@ -34,4 +34,16 @@ type Ant =
...
+    static member inline text (properties: IReactProperty list) =
+        Interop.reactApi.createElement(import "Typography.Text" "antd", createObj !!properties)
+    static member inline title (children: #seq<ReactElement>) =
+        ofImport "Typography.Title" "antd" children
+    static member inline title (properties: IReactProperty list) =
+        Interop.reactApi.createElement(import "Typography.Title" "antd", createObj !!properties)
+    static member inline paragraph (children: #seq<ReactElement>) =
+        ofImport "Typography.Paragraph" "antd" children
```
Comment (Zaid-Ajaj) [inline Feliz.AntDesign/Ant.fs]: Same as the comment above, this has to be:
```fs
ofImport "Typography.Paragraph" "antd" [ ] children
```

File: Feliz.AntDesign/Ant.fs [hunk 3]
```diff
@@ -34,4 +34,16 @@ type Ant =
...
     static member inline icon (properties: IReactProperty list) =
-        Interop.reactApi.createElement(import "Icon" "antd", createObj !!properties)
\ No newline at end of file
+        Interop.reactApi.createElement(import "Icon" "antd", createObj !!properties)
+    static member inline text (children: #seq<ReactElement>) =
+        ofImport "Typography.Text" "antd" children
+    static member inline text (properties: IReactProperty list) =
+        Interop.reactApi.createElement(import "Typography.Text" "antd", createObj !!properties)
+    static member inline title (children: #seq<ReactElement>) =
+        ofImport "Typography.Title" "antd" children
```
Comment (Zaid-Ajaj) [inline Feliz.AntDesign/Ant.fs]: ofImport "Typography.Title" "antd" [ ] children

**Review by Zaid-Ajaj — CHANGES_REQUESTED** [review]
The properties looks great, thanks a lot :heart: 

There is only one minor issue when using `ofImport` in the placement of the `children` parameter. I have added a couple of comments and how to fix them.

**Comment by Zaid-Ajaj** [comment]
On another note, why would you need `onChange` for `Ant.text`? Isn't that just for form fields?

**Comment by saidziani** [comment]
React Element importation fixed.

When it comes to `onChange`, it's available on the React API, that's why I wanted to add it.

**Comment by Zaid-Ajaj** [comment]
> When it comes to onChange, it's available on the React API, that's why I wanted to add it.

I'll check it out later, for now let us get this goodness in :smile:

**Pull request merged**